### PR TITLE
move access/auth content to reference folder, add TOC

### PR DESCRIPTION
**Pull request by Bradamant3**
Addresses issues #8260, #8538 

Redirects added, content in place for review.

See https://deploy-preview-8624--kubernetes-io-master-staging.netlify.com/docs/reference/

### CODE REVIEW

**Comment by steveperry-53** [comment]
@Bradamant3 This looks good.

Side note: I hadn't realized that all this material was under Reference. At some point in the future, let's discuss whether some of these topics belong under Concepts or Tasks.

**Comment by bradtopol** [comment]
Looks like you are making very good progress on a very big PR :-).  What I noticed is the Kubernetes is tracker on github link disappeared.  Perhaps create a page that describes what it is, what its used for, why its important, and then provide a link?  Also looked like the Open API and Swagger pages disappeared?

**Comment by Bradamant3** [comment]
One item I'll file a separate issue for: there are still files at the root of /docs/admin/ that are not included in any other section of the docs because they are so out of date. My guess is that they should be removed from master (and current release branch). Until we remove them, they are built and available at https://deploy-preview-8624--kubernetes-io-master-staging.netlify.com/docs/admin/high-availability/building/ and siblings (complete with TOC). Unlikely that anyone will trip over them, so I'm not adding any more burden to this PR.

File: content/en/docs/reference/feature-gates.md [hunk 1]
```diff
@@ -1,226 +0,0 @@
----
```
Comment (bradtopol) [inline content/en/docs/reference/feature-gates.md]: Why was this content removed?

File: content/en/docs/reference/workloads-18-19.md [hunk 2]
```diff
@@ -1,261 +0,0 @@
----
```
Comment (bradtopol) [inline content/en/docs/reference/workloads-18-19.md]: Why was this content removed?

**Comment by bradtopol** [comment]
Hi @Bradamant3 Overall this looks fantastic! I really like how you combined the Authn and Authz pages into a single list.  Those topics should definitely be co-located in that fashion. I just had a couple of questions on why you removed some content (see inline comments) and once those are answered I'm ready to approve.  Great job!

File: content/en/docs/reference/feature-gates.md (hunk 1 above)
Comment (Bradamant3) [reply content/en/docs/reference/feature-gates.md]: It's moved, not removed. New location here in the build: https://deploy-preview-8624--kubernetes-io-master-staging.netlify.com/docs/reference/command-line-tools-reference/feature-gates/
or in the repo at /content/en/docs/reference/command-line-tools-reference/feature-gates.md
(to put it where it appeared in Jekyll builds, thanks to the TOC yaml files)

File: content/en/docs/reference/workloads-18-19.md (hunk 2 above)
Comment (Bradamant3) [reply content/en/docs/reference/workloads-18-19.md]: Long out of date. We've mentioned it once or twice in sig meetings -- that it was getting to be time to yank it from docs versions where it is no longer relevant.

**Comment by Bradamant3** [comment]
@bradtopol I think I caught all your questions. lmk if not. I'll wait to rebase (among the many disadvantages of a big PR ...)

File: content/en/docs/reference/workloads-18-19.md (hunk 2 above)
Comment (bradtopol) [reply content/en/docs/reference/workloads-18-19.md]: Awesome! Sounds great!

**Comment by bradtopol** [comment]
Im good! Waiting for your rebase. Great job!

**Comment by bradtopol** [comment]
great job!
/approve
/lgtm

**Pull request merged**